addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -360,7 +360,8 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var $themes = $snippetsSideBar.find("#email_designer_themes").children();
         var $snippets = $snippetsSideBar.find(".oe_snippet");
         var selectorToKeep = '.o_we_external_history_buttons, .email_designer_top_actions';
-        $snippetsSideBar.find(`.o_we_website_top_actions>*:not(${selectorToKeep})`).hide();
+        // Overide `d-flex` class which style is `!important`
+        $snippetsSideBar.find(`.o_we_website_top_actions > *:not(${selectorToKeep})`).attr('style', 'display: none!important');
```

Comment:
Note:
1. Can't use hide as d-flex !important will prevail
2. Can't use .css() as !important not supported
3. This will wipe all `style` on the elements, but it is fine as they don't have style + we are hidding those so we don't really care about there already existing style properties

Reply:
Indeed since we are hiding it it doesn't matter, but for the record there is an option 4: you could actually use `Element.style.setProperty('display', 'none', 'important')` so you don't wipe all inline styles.